Sources/TuistSupportTesting/TestCase/TuistTestCase.swift

```diff
@@ -162,6 +162,39 @@ open class TuistTestCase: XCTestCase {
         XCTAssertFalse(output.contains(notExpected), message, file: file, line: line)
     }
 
+    public func XCTAssertDirectoryContentEqual(
```

Comment:
I'd add some simple documentation describing a little bit more what this function does - e.g. that it does not check the contents of a file recursively (which is a fine limitation)